Nuxt3のハンズオン環境が正常に動作しない

以下の環境にアクセスするとターミナルに以下のようなエラーが表示される

https://stackblitz.com/fork/github/tuqulore/vue-3-practices/tree/main/handson-nuxt?file=app.vue&terminal=dev
https://stackblitz.com/fork/github/tuqulore/vue-3-practices/tree/main/handson-nuxt-playground?file=app.vue&terminal=dev

```
~/projects/tuqulore-vue-3-practices-uize7u
❯ npm install && npm run dev
┌ [1/4] 🔍  Resolving dependencies
Nuxt CLI v3.0.0-rc.4-27588443.cf25525                                                    10:11:52
                                                                                         10:11:52
  > Local:    http://localhost:3000// 
  > Network:  http://192.168.1.104:3000//


 ERROR  Cannot start nuxt:  Cannot find module 'nuxt'                                    10:11:53
Require stack:
- /home/projects/tuqulore-vue-3-practices-uize7u/index.js

  Require stack:
  - index.js
  at InternalError (https://tuqulore-vue-3-practices-uize7u.w-corp.staticblitz.com/blitz.c4393a56cbb8e2fec13bc31575603ce9568f347d.js:6:291603)
  at Module._resolveFilename (https://tuqulore-vue-3-practices-uize7u.w-corp.staticblitz.com/blitz.c4393a56cbb8e2fec13bc31575603ce9568f347d.js:6:217231)
  at resolve (https://tuqulore-vue-3-practices-uize7u.w-corp.staticblitz.com/blitz.c4393a56cbb8e2fec13bc31575603ce9568f347d.js:6:415201)
  at _resolve (node_modules/jiti/dist/jiti.js:1:108295)
  at resolveModule@node_modules/ (nuxt/kit/dist/index.mjs:316:36)
  at importModule@node_modules/ (nuxt/kit/dist/index.mjs:339:37)
  at loadNuxt@node_modules/ (nuxt/kit/dist/index.mjs:735:43)
  at <anonymous> (<anonymous>)

```

--- a/handson-nuxt-playground/nuxt.config.js
+++ b/handson-nuxt-playground/nuxt.config.js
@@ -1,4 +1,4 @@
-import { defineNuxtConfig } from 'nuxt3'
+import { defineNuxtConfig } from 'nuxt'
 
 // https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
 export default defineNuxtConfig({

--- a/handson-nuxt-playground/package.json
+++ b/handson-nuxt-playground/package.json
@@ -7,7 +7,7 @@
     "url": "https://github.com/tuqulore/vue-3-practices/issues"
   },
   "devDependencies": {
-    "nuxt3": "3.0.0-rc.4-27588443.cf25525"
+    "nuxt": "3.0.0-rc.4"
   },
   "homepage": "https://github.com/tuqulore/vue-3-practices#readme",
   "keywords": [

--- a/handson-nuxt/nuxt.config.js
+++ b/handson-nuxt/nuxt.config.js
@@ -1,4 +1,4 @@
-import { defineNuxtConfig } from 'nuxt3'
+import { defineNuxtConfig } from 'nuxt'
 
 // https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
 export default defineNuxtConfig({

--- a/handson-nuxt/package.json
+++ b/handson-nuxt/package.json
@@ -7,7 +7,7 @@
     "url": "https://github.com/tuqulore/vue-3-practices/issues"
   },
   "devDependencies": {
-    "nuxt3": "3.0.0-rc.4-27588443.cf25525"
+    "nuxt": "3.0.0-rc.4"
   },
   "homepage": "https://github.com/tuqulore/vue-3-practices#readme",
   "keywords": [

